src/components/README.md

```diff
@@ -0,0 +1,17 @@
+# Components directory
+
+To allow for pre-compilation, the creation of "helper" components is delayed until
+after Mimi is loaded, via the `__init__()` function in Mimi.jl.
+All `jl` files in this directory are loaded by `__init__()`. Additions to this 
+directory should be documented here.
+
+*Note that components loaded by __init__() are loaded by the `Main` module, not the `Mimi`
+module. So to place components defined here in `Mimi`, prefix the component names with 
+`Mimi.`*
```

Comment:
Apparently, __init__ is run after the module is loaded, meaning back in whatever module loaded Mimi. I view this as a bug since it violates the "principle of least astonishment".